From 3ef416e1a2c9338dda552586abbbb8505e95ff49 Mon Sep 17 00:00:00 2001
From: Martin Braun <martin.braun@ettus.com>
Date: Mon, 22 Aug 2016 14:40:03 -0700
Subject: [PATCH 42/42] Preparing branch for 3.9.5 Release

- Updated CHANGELOG (incl. Debian)
- Updated version
- Updated images package

FPGA submodule pointer already up-to-date.
---
 CHANGELOG                           | 15 +++++++++++++++
 host/CMakeLists.txt                 |  4 ++--
 host/cmake/Modules/UHDVersion.cmake |  2 +-
 host/cmake/debian/changelog         | 18 ++++++++++++++++++
 4 files changed, 36 insertions(+), 3 deletions(-)

diff --git a/CHANGELOG b/CHANGELOG
index 9cc22f4..863363f 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -1,6 +1,21 @@
 Change Log for Releases
 ==============================
 
+## 003.009.005
+- B200: Update DSPs after changing tick rate
+- X300: Added option to disable ADC self test, prevent DAC from
+  underrunning
+- UBX: Fixed noise issues, reduced power consumption/heat, added codes for
+  upcoming board revisions, force RX LNAs on (reduces rx settling time)
+- WBX: Fixed rev2 and rev3 boards on X300
+- Utils/Examples: Cleaned up query_gpsdo_sensors, test_dboard_coercion,
+- Manual: Minor fixes and updates
+- CMake: Fix lib64 detection, better platform detection, Doxygen may use shorter
+  filenames
+- Octoclock: GPSDO-related fixes, sequence number consistency fixes, UART
+  fixes (off-by-one errors). uhd_usrp_probe will pick up an OC now,
+  added Wireshark dissector for OC packets
+
 ## 003.009.004
 - GPIO control: Fix address mismatch for RX and full duplex.
   This fixes full-duplex mode for most devices.
diff --git a/host/CMakeLists.txt b/host/CMakeLists.txt
index 9534dd7..b53c789 100644
--- a/host/CMakeLists.txt
+++ b/host/CMakeLists.txt
@@ -278,8 +278,8 @@ UHD_INSTALL(FILES
 #{{{IMG_SECTION
 # This section is written automatically by /images/create_imgs_package.py
 # Any manual changes in here will be overwritten.
-SET(UHD_IMAGES_MD5SUM "c33b36a74d82198c8cfede64d8775c46")
-SET(UHD_IMAGES_DOWNLOAD_SRC "uhd-images_003.009.004-release.zip")
+SET(UHD_IMAGES_MD5SUM "e78047f58fde583d1527abbbbb156c71")
+SET(UHD_IMAGES_DOWNLOAD_SRC "uhd-images_003.009.005-rc1.zip")
 #}}}
 
 ########################################################################
diff --git a/host/cmake/Modules/UHDVersion.cmake b/host/cmake/Modules/UHDVersion.cmake
index 4b26efd..ca00307 100644
--- a/host/cmake/Modules/UHDVersion.cmake
+++ b/host/cmake/Modules/UHDVersion.cmake
@@ -28,7 +28,7 @@ FIND_PACKAGE(Git QUIET)
 ########################################################################
 SET(UHD_VERSION_MAJOR 003)
 SET(UHD_VERSION_MINOR 009)
-SET(UHD_VERSION_PATCH 004)
+SET(UHD_VERSION_PATCH 005)
 SET(UHD_VERSION_DEVEL FALSE)
 
 ########################################################################
diff --git a/host/cmake/debian/changelog b/host/cmake/debian/changelog
index f83d834..6b50f5b 100644
--- a/host/cmake/debian/changelog
+++ b/host/cmake/debian/changelog
@@ -1,3 +1,21 @@
+uhd (3.9.5-0ubuntu1) trusty; urgency=low
+
+  - B200: Update DSPs after changing tick rate
+  - X300: Added option to disable ADC self test, prevent DAC from
+    underrunning
+  - UBX: Fixed noise issues, reduced power consumption/heat, added codes for
+    upcoming board revisions, force RX LNAs on (reduces rx settling time)
+  - WBX: Fixed rev2 and rev3 boards on X300
+  - Utils/Examples: Cleaned up query_gpsdo_sensors, test_dboard_coercion,
+  - Manual: Minor fixes and updates
+  - CMake: Fix lib64 detection, better platform detection, Doxygen may use shorter
+    filenames
+  - Octoclock: GPSDO-related fixes, sequence number consistency fixes, UART
+    fixes (off-by-one errors). uhd_usrp_probe will pick up an OC now,
+    added Wireshark dissector for OC packets
+
+ -- Ettus Research <packages@ettus.com>  Mon, 22 Aug 2016 02:36:40 -0800
+
 uhd (3.9.4-0ubuntu1) trusty; urgency=low
 
   - GPIO control: Fix address mismatch for RX and full duplex.
-- 
2.1.4

